midealocal/devices/ac/message.py

```
@@ -1271,7 +1356,20 @@ def _parse_capabilities(self, params: dict[int, bytearray]) -> None:
             caps["display_control"] = value in B5_DISPLAY_VALUES
         if NewProtocolTags.b5_electricity in params:
             value = params[NewProtocolTags.b5_electricity][0]
-            caps["rate_select"] = value > B5_ELECTRICITY_UNSUPPORTED_VALUE
+            caps["energy_stats"] = value in B5_ENERGY_STATS_VALUES
```

Ruff (0.16.3)

[error] Undefined name `B5_ENERGY_STATS_VALUES`

(F821)